SEED-platform/seed
# Filter matching

#### Any background context you want to provide?
#### What's this PR do?
1) Creates merged coverage from python and functional tests. Increases run time of functional tests by 5-7 mins. Could reduce that if we went to a different coverage tool than coveralls (like codecov). 
2) Updates to filter and sorting for hand matching detail and pairing pages. Also updates to css for pairing page.
3) Functional tests for (2)
#### How should this be manually tested?
See coverall reports for combined coverage. See matching detail and pairing pages for filter and sort updates. 
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Definition of Done:
